src/test/regress/expected/multi_view.out

```diff
@@ -206,6 +207,35 @@ SELECT count(*) FROM temp_lineitem;
   1706
 (1 row)
 
+-- can create and query materialized views
+CREATE MATERIALIZED VIEW mode_counts
+AS SELECT l_shipmode, count(*) FROM temp_lineitem GROUP BY l_shipmode;
+SELECT * FROM mode_counts WHERE l_shipmode = 'AIR' ORDER BY 2 DESC, 1 LIMIT 10;
+ l_shipmode | count 
+------------+-------
+ AIR        |  1706
+(1 row)
+
+-- materialized views are local, cannot join with distributed tables
```

Comment:
Ah, we might hit `ERROR:  relation % is not distributed` when we use mt views with joins. 

```SQL
CREATE MATERIALIZED VIEW some_users AS SELECT * FROM users_table WHERE value_1 < 3;
SELECT 45
select distinct some_users.user_id from some_users, events_table WHERE event_type IN (1,2,23) AND events_table.user_id = some_users.user_id  ;
ERROR:  relation some_users is not distributed
```

I think that's not a blocker for merging this in. Just noting in case anyone hits it. (Same example with regular view works fine.)